cumulusci/tasks/metadata_etl/objects.py

```diff
@@ -0,0 +1,65 @@
+from cumulusci.core.exceptions import TaskOptionsError
+from cumulusci.core.utils import process_list_arg
+from cumulusci.tasks.metadata_etl import MetadataSingleEntityTransformTask
+from cumulusci.utils.xml.metadata_tree import MetadataElement
+
+OBJ_SETTINGS = [
+    "Activities",
+    "BulkApi",
+    "Feeds",
+    "History",
+    "Licensing",
+    "Reports",
+    "Search",
+    "Sharing",
+    "StreamingApi",
+]
+OBJ_SETTINGS_STR = ", ".join(OBJ_SETTINGS)
+
+
+class SetObjectSettings(MetadataSingleEntityTransformTask):
+    entity = "CustomObject"
+    task_options = {
+        "enable": {
+            "description": f"Array of object settings to enable. Uses the setting name.  Available values: {OBJ_SETTINGS_STR}"
+        },
+        "disable": {
+            "description": f"Array of object settings to disable. Uses the setting name.  Available values: {OBJ_SETTINGS_STR}"
+        },
+        # Only include api_names option since namespace injection isn't supported
```

Comment:
Let's get namespace injection supported here - it could be needed for projects that wish to mutate their own settings.

Reply:
There's an example of supporting injection on the `CustomObject` entity in `SetOrgWideDefaults`.

Reply:
OK, I wasn't sure if there was a good use case for modifying the project's objects via this task.  I guess I could see a case where you want, for example, history disabled in the packaged object but then a solution configuration might want to enable it.

I'll get that added in and tested.

Reply:
I think this was easier than I thought.  Since the elements in the object aren't ever namespaced, I think the base class handles namespace injection in the api_names for me automatically.  I just added those options back.